src/System.IO.Compression/ref/4.1.0/System.IO.Compression.depproj

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <AssemblyVersion>4.1.2.0</AssemblyVersion>
```

Comment:
I believe this doesn't really matter, because the actual assembly is downloaded from the package, this is just to match what the ref version was before.